functions

```diff
@@ -306,3 +306,23 @@ function pull_easyengine_images() {
   ee_log_info1 "Pulling EasyEngine docker images"
   "$EE4_BINARY" cli info
 }
+
+function add_ssl_renew_cron() {
+  # noglob is required to ensure that '*' is not expanded to list all files in current directory while printing SSL_RENEW_CRON
+  set -o noglob
+  
+  SSL_RENEW_CRON="PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin\n\n0 0 * * * /usr/local/bin/ee site ssl-renew --all >> /opt/easyengine/logs/cron.log 2>&1"
```

Comment:
Take the path value dynamically.

```suggestion
  SSL_RENEW_CRON="PATH=$PATH\n\n0 0 * * * /usr/local/bin/ee site ssl-renew --all >> /opt/easyengine/logs/cron.log 2>&1"

Reply:
Also check if `/usr/local/bin` is there in path or not. If it is not there, then add it.